packages/recomponents/src/components/r-tabs/id-helpers.js

```diff
@@ -0,0 +1,7 @@
+export function generateControlId(panelId) {
+    return `${panelId}-control`;
+}
+
+export function generatePanelId(uid) {
+    return `tab-${uid}`;
+}
```

Comment:
How about converting those so that function names match results? The most typical flow can be used so it's `tab` that controls `tabpanel`.
Looking only at those methods it's to understand what's happening and a person needs to read (and process) the entire code for tabs.

Reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

Reply:
I like that naming convention - it's definitely more readable! :)
I've refactored labelling and `id-helpers.js` 🚀